docs/application/native/guides/data/data-storages.md

```diff
@@ -17,11 +17,25 @@ The main features of the Storage API include:
 
 ## Prerequisites
 
-To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+To enable your application to use the storage functionality:
 
-```
-#include <storage.h>
-```
+1. To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+
+   ```
+   #include <storage.h>
+   ```
+
+2. To access the files or directories of the device storage, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+   ```
+   <privileges>
+      <!--For the internal storage-->
```

Comment:
Only one more point that I almost missed.

Please use 4 spaces for the indentation.
We need 1 more space from line 31 to line 34.

Reply:
@mijin85cho I updated about this. please review it again. Thank you.